fvm/env.go

```diff
@@ -35,6 +37,15 @@ type hostEnv struct {
 	rng            *rand.Rand
 }
 
+func (e *hostEnv) Hash(data []byte, hashAlgorithm string) []byte {
```

Comment:
@turbolent this came with the new version of cadence. I hope I implemented it as intended.

Reply:
Yep, this looks great!